package.json

```diff
@@ -24,23 +24,23 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@mui/material": "5.6.x",
-    "@mui/styled-engine-sc": "^5.6.x",
-    "@mui/styles": "5.6.x",
+    "@mui/material": "5.8.x",
+    "@mui/styled-engine-sc": "^5.8.x",
+    "@mui/styles": "5.8.x",
     "connected-react-router": "6.x.x",
     "history": "^4.10.1",
     "immer": "9.x.x",
     "normalizr": "3.x.x",
-    "prop-types": "^15.7.2",
+    "prop-types": "^15.8.1",
     "react": "17.x.x",
     "react-dom": "^17.0.2",
     "react-error-boundary": "3.x.x",
     "react-hot-loader": "4.x.x",
     "react-redux": "7.x.x",
     "react-router-dom": "5.x.x",
-    "redux": "^4.1.0",
+    "redux": "^4.2.0",
```

Comment:
I see that this release deprecates `createStore` I think we still use that. Seems like they are trying to push people toward Redux Toolkit. Sorry if this has been discussed already 😅.